trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/parasite_total.py

```diff
@@ -61,6 +61,13 @@ def parasite_total(state,settings,geometry):
         conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].parasite_drag_coefficient  = parasite_drag * ref_area/vehicle_reference_area * propulsor.number_of_engines
         total_parasite_drag += parasite_drag * ref_area/vehicle_reference_area * propulsor.number_of_engines
 
+    # from pylons
```

Comment:
There isn't a pylon script for supersonics. I guess the comment I had in there isn't accurate. This should be more of a reminder that the component is missing (I had also been using it for crosschecking results with fidelity zero). I do agree with the sentiment, and it's part of the updates I'm planning to continue with on this branch.